site/src/pages/WorkspaceSettingsPage/WorkspaceSettingsForm.tsx

```diff
@@ -106,7 +106,11 @@ export const WorkspaceSettingsForm: FC<{
           </FormFields>
         </FormSection>
       )}
-      <FormFooter onCancel={onCancel} isLoading={form.isSubmitting} />
+      <FormFooter
+        onCancel={onCancel}
+        isLoading={form.isSubmitting}
+        submitDisabled={!templatePoliciesEnabled && !workspace.allow_renames}
```

Comment:
I'd like to keep the structure of the form, since it can optionally be expanded to include template policies if those are enabled. Also I don't love the idea of conditionally having items in a sidebar, which would be the case here if we removed it from the sidebar (because there's nothing to show). 

When you mean remove the form entirely, do you mean just the form footer buttons, or actually the whole page? Example of the former.
<img width="1383" alt="Screenshot 2023-12-15 at 9 05 31 AM" src="https://github.com/coder/coder/assets/19379394/32034d10-9fb3-45f6-84aa-7a97b62387b3">

Reply:
For comparison with the template cpolicies enabled:
<img width="1390" alt="image" src="https://github.com/coder/coder/assets/19379394/812fc669-e905-4325-900d-942fd4276866">

Reply:
Something like 
```  const formEnabled = (templatePoliciesEnabled && !workspace.template_require_active_version)
    || workspace.allow_renames```